Release notes updates (#65497)
Azure/azure-content: articles/active-directory-b2c/page-layout.md

Page layout packages are periodically updated to include fixes and improvements in their page elements. The following change log specifies the changes introduced in each version.

## Self-asserted page (selfasserted)

**2.1.1**

> - Added a UXString `heading` in addtion to `intro` to show on page as a title, it will be hidden by default.  

Reviewer: ```suggestion
- You can now use the `heading` and `intro` HTML elements to display a title on the page. These elements are hidden by default.  
```

> - Added a UXString `heading` in addtion to `intro` to show on page as a title, it will be hidden by default.  

Reviewer 2: I recommend the following to make this clearer:
Added a UXString heading in addtion to intro to show on page as a title. This will be hidden by default.

> - Added a UXString `heading` in addtion to `intro` to show on page as a title. This will be hidden by default.

Reviewer: ```suggestion
- Added a UXString `heading` in addition to `intro` to display on the page as a title. This is hidden by default.
```

> - Allow saving password to iOS keychain.

Reviewer: ```suggestion
- Added support for saving passwords to iCloud Keychain.
```

> - Allow picking DOM layout via policy or querystring parameter `pageFlavor`: classic, oceanBlue, slateGray.

Reviewer: ```suggestion
- Added support for using policy or the QueryString parameter `pageFlavor` to select the layout (classic, oceanBlue, or slateGray).
```

- Added disclaimers on self-asserted page.

> - Focus will be placed on the first editable field on page load.

Reviewer: ```suggestion
- Focus is now placed on the first editable field when the page loads.
```

> - Focus will be placed on the first error field when multiple fields are errored.

Reviewer: ```suggestion
- Focus is now placed on the first error field when multiple fields have errors.
```

> - Focus will be placed on 'change' button once email verification code is verified.

Reviewer: ```suggestion
- Focus is now placed on the 'change' button after the email verification code is verified.
```

- The username/email and password fields now use the `form` HTML element to allow Edge and Internet Explorer (IE) to properly save this information.
- Added a configurable user input validation delay for improved user experience.

> - Fixed an accessibility issue that error message won't be read by narrator. 

Reviewer 2: "Fixed an accessibility issue where error message won't be read by narrator."

> - Fixed an accessibility issue where error message won't be read by narrator.

Reviewer: ```suggestion
- Fixed an accessibility issue so that error messages are now read by Narrator. 
```

> - Put focus on password field when email is verified. 

Reviewer: ```suggestion
- Focus is now placed on the password field after the email is verified.
```

> - Removed `autofocus` from checkbox control. 

Reviewer: ```suggestion
- Removed `autofocus` from the checkbox control. 
```

> - Support display widget for phone number verification. 

Reviewer: ```suggestion
- Added support for a display widget for phone number verification.
```

## Unified sign-in sign-up page with password reset link (unifiedssp)

**2.1.1**

> - Added a UXString `heading` in addtion to `intro` to show on page as a title. This will be hidden by default.

Reviewer: ```suggestion
- Added a UXString `heading` in addition to `intro` to display on the page as a title. This is hidden by default.
```

> - Allow picking DOM layout via policy or querystring parameter `pageFlavor`: classic, oceanBlue, slateGray.

Reviewer: ```suggestion
- Added support for using policy or the QueryString parameter `pageFlavor` to select the layout (classic, oceanBlue, or slateGray).
```

> - Added a UXString `heading` in addtion to `intro` to show on page as a title, it will be hidden by default.  

Reviewer 2: "Added a UXString heading in addtion to intro to show on page as a title. This will be hidden by default."

> - Allow using password from iOS Keychain.

Reviewer: ```suggestion
- Added support for saving passwords to iCloud Keychain.
```

> - Focus will be placed on the first error field when multiple fields are errored.

Reviewer: ```suggestion
- Focus is now placed on the first error field when multiple fields have errors.
```

> - Focus will be placed on the first editable field on page load.

Reviewer: ```suggestion
- Focus is now placed on the first editable field when the page loads.
```

> - Added new location for claims provider selection link `bottomUnderFormClaimsProviderSelections`.

Reviewer: ```suggestion
- Added a new location for the claims provider selection link `bottomUnderFormClaimsProviderSelections`.
```

> - Deprecated UXStrings clean up.

Reviewer 2: "Removed UXStrings that are no longer used."

## MFA page (multifactor)

**1.2.2**

> - Fixed issue appeared on iOS that verification code can not be autofilled. 

Reviewer: ```suggestion
- Fixed an issue with auto-filling the verification code when using iOS.
```

> - Fixed issue appeared on Android that token can't be redirect to RP from a Android Webview. 

Reviewer: ```suggestion
- Fixed an issue with redirecting a token to the relying party from Android Webview. 
```

> - Added a UXString `heading` in addtion to `intro` to show on page as a title, it will be hidden by default.  

Reviewer: ```suggestion
- Added a UXString `heading` in addition to `intro` to display on the page as a title. This is hidden by default.  
```

> - Allow picking DOM layout via policy or querystring parameter `pageFlavor`: classic, oceanBlue, slateGray.

Reviewer: ```suggestion
- Added support for using policy or the QueryString parameter `pageFlavor` to select the layout (classic, oceanBlue, or slateGray).
```
